source/features/conversation-activity-filter.tsx

```diff
@@ -152,6 +152,10 @@ async function addWidget(header: string): Promise<void> {
 }
 
 async function init(): Promise<void> {
+	// Reset the dropdown state #4997
+	currentSetting = 'default';
+	select('.repository-content')!.classList.remove('rgh-conversation-activity-is-filtered');
```

Comment:
Alternatively this could be a `deinit` function

Reply:
Which way is safer?

Reply:
This one I guess? Since it's inside the `init`, it's guaranteed to run before the widgets are created. Since `deinit` is essentially a callback, it *could* fail to run for whatever reason.